src/embind/embind.js

```diff
@@ -1913,11 +1916,15 @@ var LibraryEmbind = {
   ) {
     var rawArgTypes = heap32VectorToArray(argCount, rawArgTypesAddr);
     methodName = readLatin1String(methodName);
+    var humanName = methodName;
+    if(methodName == "@@iterator"){
+        methodName = Symbol.iterator;
```

Comment:
I don't know much about ES6 iterators, but this seems wrong.  The methodName is used to construct the name of the function, right?